### DISCUSSION TITLE
What is a clean way to add timestamps and `_quarto.yml` information in a filename?

### DISCUSSION THREAD

**Discussion by liyongg** [quarto-dev/quarto-cli#5233] [Q&A] [answered]
In my Quarto book project, I am trying to add a timestamp and additional information to the filename of the rendered output based on the project `_quarto.yml`. However, from #606 I assumed that it would not be possible in the `output-file` tag under `book`. 

Now I have written a script and tagged it in `post-render` under `project`. Is this the correct way (as of now) for what I am trying to achieve?

**Comment by mcanouil** [comment]
Simply use date and the formatting feature of Quarto, see <https://quarto.org/docs/reference/dates.html>.

**Reply by cderv** [reply, marked as answer]
Will this really allow to write date information to output filename  ? It is dedicated to tweak the date information for output document content, isn't it ? 

it seems to me the post render rendered script is the way to go to had some information into the output filename of a rendering.

**Reply by mcanouil** [reply]
Oh, I misread "in filename" as "in file".

Indeed, the only way to add a timestamp and/or rename/move rendered files is using post-render script (<https://quarto.org/docs/projects/scripts.html#pre-and-post-render>).

**Reply by liyongg** [reply]
Thank you! Good to know that the post-render script was not a waste of time (;
